packages/shared/components/popups/SendConfirmationPopup.svelte

```diff
@@ -80,6 +74,18 @@
         type: ActivityType.Transaction,
     }
 
+    $: initialExpirationDate = getInitialExpirationDate()
```

Comment:
Not sure if this needs to be reactive since it's just an initialization. I think you'll have to add a date function parameter then pass `expirationDate` to be reactive to that variable, iiuc. Should this just be a regular variable?

Reply:
You're right, this doesn't need to be reactive